### ISSUE TITLE
3. WAVE Accessibility issues 

### ISSUE THREAD

**Issue by IngerMathildeEiklund** [IngerMathildeEiklund/exam-project-1#80]
Homepage: 
 
WAVE claims the text on the hero image is too low, however I have added aria-hidden="true" so it should be fine. 

One product page: 

Same thing as earlier, notes a too low contrast on the user image, and empty alt text, but the whole image has aria-hidden="true".

Cart page: 

"Multiple form labels" on the quantity inputs. For each time a product that increases the cart.length and another quantity input appears it has the same id as all the others. The fix:

~~[x] Add aria-labelledby="" to the label as multiple form labels are neccessary.~~

This was a little more work than I had thought, see sub-issue under. 

Checkout page: 

Passed.

Successpage: 

Passed. 

Footer pages: 

Passed.
